configure.ac

```diff
@@ -70,6 +70,7 @@ AC_ARG_ENABLE([direct],
 
 dnl Checks for programs
 AC_PROG_CC_C99
+AM_PROG_CC_C_O
```

Comment:
Which version of automake are we recommending? `AM_PROG_CC_C_O` is obsolescent in newer versions and automake will print a warning about the configury containing the macro.